simpleclient/src/main/java/io/prometheus/client/Gauge.java

```diff
@@ -241,6 +261,16 @@ public Timer startTimer() {
     return noLabelsChild.startTimer();
   }
 
+  /**
+   * Executes runnable code (ie a Java 8 Lambda) and observes a duration of how long it took to run.
+   *
+   * @param timeable Code that is being timed
+   * @return Measured duration in seconds for timeable to complete.
+   */
+  public double setToTime(Runnable timeable){
+    return noLabelsChild.setToTime(timeable);
```

Comment:
Okay, can you add them to Summary and Histogram too?

"i.e." still needs changing in the comment